cmd/gcp-controller-manager/app/gke_approver.go

```diff
@@ -171,6 +173,7 @@ type csrCheckFunc func(opts approverOptions, csr *capi.CertificateSigningRequest
 
 type csrValidator struct {
 	name       string
+	metricName string
```

Comment:
flowLabel, authFlowLabel, authorizationFlowLabel would all be more descriptive.

Reply:
Done